octobot_commons/configuration/user_inputs.py

```diff
@@ -0,0 +1,193 @@
+# pylint: disable=W1203
+#  Drakkar-Software OctoBot-Commons
+#  Copyright (c) Drakkar-Software, All rights reserved.
+#
+#  This library is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU Lesser General Public
+#  License as published by the Free Software Foundation; either
+#  version 3.0 of the License, or (at your option) any later version.
+#
+#  This library is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+#  Lesser General Public License for more details.
+#
+#  You should have received a copy of the GNU Lesser General Public
+#  License along with this library.
+import octobot_commons.enums as enums
+
+
+async def user_input(
+    name,
+    input_type,
+    def_val,
+    tentacle_type,
+    tentacle_name,
+    saved_config,
+    run_data_writer,
+    min_val=None,
+    max_val=None,
+    options=None,
+    title=None,
+    item_title=None,
+    other_schema_values=None,
+    editor_options=None,
+    read_only=False,
+    is_nested_config=None,
+    nested_tentacle=None,
+    parent_input_name=None,
+    show_in_summary=True,
+    show_in_optimizer=True,
+    path=None,
+    order=None,
+    flush_if_necessary=False,
+    skip_flush=False,
+):
```

Comment:
I was wondering if a class `UserInput` couldn't be better to prevent having a huge number of args.